discovery-provider/Dockerfile

```diff
@@ -56,19 +56,19 @@ RUN echo 'http://mirror.leaseweb.com/alpine/v3.13/community' >> /etc/apk/reposit
     mkdir /usr/local/openresty/logs
 
 # versions
-# alpine-sdk 1.0-r0
```

Comment:
-1 on having these versions in the comments here

If we really want something like this, we should add it to line 11.